src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -1229,10 +1231,6 @@ public void GetPartsOfAssignmentExpressionOrStatement(
         right = assignment.Right;
     }
 
-    // C# does not have assignment statements.
-    public bool IsAnyAssignmentStatement([NotNullWhen(true)] SyntaxNode? node)
-        => false;
```

Comment:
this dichotomy didn't make sense.  we can't support IsSimpleAssignmetnStatemnet but then say we don't support any assignment statement. 

Audited all usages of this call to make sure they're fine with a C# impl now.